docs/api-and-integration/teamcity.md

```diff
@@ -17,16 +17,10 @@ While Octopus Deploy will be used to take those NuGet packages and to push them
 Integration with TeamCity involves two major parts:
 
 1. Creating the NuGet packages from your applications
-   2. Optionally, when a build completes, having TeamCity make requests to your Octopus Server to:
-       (a) Create releases; (b) Trigger deployments and/or (c) Promote releases when a build completes
-
-- Installing the plugin
-- Creating Octopus-compatible NuGet packages using TeamCity
-- Using Octopus as a Package Repository
-- Consuming the TeamCity NuGet feed in Octopus
-- Creating and pushing packages from TeamCity to Octopus
-- Triggering deployments from TeamCity
-- Using the plugin with Linux build agents
+2. Optionally, when a build completes, having TeamCity make requests to your Octopus Server to:
+   (a) Create releases;
```

Comment:
I'm trying to keep the readme instructions up to date, if there is anything you think it would be good to mention let me know or send a PR 😄